Tests/govuk_ios/govuk_ios_unit_tests/Specs/ViewModels/SettingsViewModelTests.swift

```diff
@@ -45,13 +45,13 @@ struct SettingsViewModelTests {
 
         let linkSection = sut.listContent[2]
         #expect(linkSection.rows[0].title == "Privacy policy")
-        #expect(linkSection.rows[1].title == "Open source licences")
+        #expect(linkSection.rows[1].title == "Accessibility statement")
+        #expect(linkSection.rows[2].title == "Open source licences")
+        #expect(linkSection.rows[3].title == "Terms and conditions")
     }
     
     @Test
     func analytics_toggledOnThenOff_deniesPermissions() throws {
-        #expect(sut.listContent.count == 3)
```

Comment:
I am answering on Pete's behalf, but I assume the requirement to change a tonne of unit tests because the number of sections changed was a pain.
This unit test tests the switch action, so checking if it doesn't add/ remove sections is probably a little unnecessary.